.github/workflows/ci.yml

```diff
@@ -19,41 +19,28 @@ jobs:
         version:
           - 'nightly'
           - '1'
-          - '1.6'
-        os:
-          - ubuntu-latest
-          - macos-latest
-          - windows-latest
-        arch:
-          - x64
+          - '1.6'  # lowest declared julia compat in `Project.toml`
```

Comment:
Do we need these changes for this PR? Otherwise I think a separate PR for upgrading the ci.yml file might be more appropriate, as this allows us better to spot what has changed when writing release notes.

Reply:
Dropped workflows changes.